TASVideos/Extensions/EntityExtensions.cs

```diff
@@ -269,7 +269,7 @@ public static IQueryable<RoleDisplayModel> ToRoleDisplayModel(this IQueryable<Ro
 		});
 	}
 
-	public static IQueryable<UserFileModel> ToUserFileModel(this IQueryable<UserFile> userFiles)
+	public static IQueryable<UserFileModel> ToUserFileModel(this IQueryable<UserFile> userFiles, bool hideComments = true)
```

Comment:
There's a better way to do this, but I'll approve and merge.

For future reference you can declare this boolean in the page model and the view will have access to it